src/data_transfer/models/pod_data.go

```diff
@@ -0,0 +1,96 @@
+package models
+
+import (
+	"log"
+	"strconv"
+	"strings"
+
+	excelAdapterModels "github.com/HyperloopUPV-H8/Backend-H8/excel_adapter/models"
+)
+
+type PodData struct {
+	Boards map[string]Board `json:"boards"`
+}
+
+func (podData *PodData) AddPacket(board string, ip string, desc excelAdapterModels.Description, values []excelAdapterModels.Value) {
+	if podData.Boards == nil {
+		podData.Boards = make(map[string]Board)
+	}
+
+	id, err := strconv.ParseUint(desc.ID, 10, 16)
+	if err != nil {
+		log.Fatalf("data transfer: AddPacket: %s\n", err)
+	}
+
+	dataBoard, ok := podData.Boards[board]
```

Comment:
No boards should be defined at first, so no need to check

Reply:
Boards is defined, here I'm checking if the board of the packet exsist (which may not be the case) and adding a new entry or updating the existing one (we cannot init this beforehand because with this approach we don't have a board list in main)